oauth-proxy/oauthHandlers/revokeUserGrantHandler.js

```
@@ -82,8 +93,13 @@ const deleteGrantsOnClientAndUserId = async (oktaClient, userId, clientId) => {
   return retValue;
 };
 
-const checkForValidParams = async (oktaClient, config, clientId, email) => {
-  if (!config.enable_okta_consent_endpoint) {
+const checkForValidParams = async (
+  oktaClient,
+  enable_consent_endpoint,
+  clientId,
+  email
+) => {
+  if (!enable_consent_endpoint) {
     throw {
       status: 403,
       errorMessage: "Revoking grants is disabled in this environment.",
```

This error message isn't accurate anymore... it's not just environment based, it's also authorization route based.

There was a recommendation on Luke's management_endpoint PR to change where this check is done that I think we should also apply here. Instead of passing `enable_consent_endpoint` through multiple layers, let's check it in index.js and let the framework throw a 404 if it's not enabled.

```
  if (enable_consent_endpoint) {
    router.delete(api_category + app_routes.grants, async (req, res, next) => {
      await oauthHandlers
        .revokeUserGrantHandler(okta_client, req, res, next)
        .catch(next);
    });
  }
```

This also satisfies the 404 requirement in the story.